docs/user-guide/index.md

```diff
@@ -57,6 +57,9 @@ the different pieces:
 * [Connectors](#connectors)
 * [Formatters](#formatters)
 
+## Permission Issue
```

Comment:
Add empty line after this line

(see also: https://travis-ci.org/sonarwhal/sonar/jobs/250504088#L2417)